packages/formatter-codeframe/src/formatter.ts

```diff
@@ -160,6 +159,6 @@ export default class CodeframeFormatter implements IFormatter {
 
         const color: typeof chalk = totalErrors > 0 ? chalk.red : chalk.yellow;
 
-        logger.log(color.bold(`${logSymbols.error} Found a total of ${totalErrors} ${pluralize('error', totalErrors)} and ${totalWarnings} ${pluralize('warning', totalWarnings)}`));
+        logger.log(color.bold(`${logSymbols.error} Found a total of ${totalErrors} ${totalErrors ? 'errors' : 'error'} and ${totalWarnings} ${totalWarnings !== 1 ? 'warnings' : 'warnings'}`));
```

Comment:
You are using `'warnings'` in both expressions.

Also for all the conditions I think we should use
```
=== 1 ? 'warning' : 'warnings' // or equivalent
```
instead of 
```
!== 1 ? 'warnings' : 'warning'
```

The reason is that dealing with negations is overhead that we can avoid. It took me a couple seconds to know exactly what expression was executed in each case and it would have been straightforward the other way.

Reply:
(Plural was usually the more common case, so it was a micro optimization.)

I changed things to `=== 1`.